rootfs/templates/kops/default.yaml

```diff
@@ -50,6 +51,8 @@ spec:
     - ResourceQuota
     - NodeRestriction
     - Priority
+    - Initializers
+    - DenyEscalatingExec
```

Comment:
I suspect this might cause problems with the teleport node installation via helm and the ability to call `systemctl restart`

Reply:
It might cause problems, but I prefer to put it in and see if we can find a better solution to those problems.